tensorflow/core/kernels/mkl/mkl_batch_matmul_op.cc

```diff
@@ -35,8 +35,10 @@ limitations under the License.
 #include "tensorflow/core/kernels/matmul_op_impl.h"
 #include "tensorflow/core/kernels/mkl/mkl_batch_matmul_helper.h"
 #include "tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h"
+#include "tensorflow/core/platform/errors.h"
 #include "tensorflow/core/platform/types.h"
 #include "tensorflow/core/util/matmul_bcast.h"
+#include "third_party/eigen3/unsupported/Eigen/CXX11/Tensor"
```

Comment:
I don't think this header exists anymore - it should just be "unsupported/Eigen..."

Reply:
Corrected.